src/commcare_cloud/terraform/modules/network/main.tf

```diff
@@ -327,11 +327,15 @@ resource "aws_security_group" "rds" {
   vpc_id = "${aws_vpc.main.id}"
 
   ingress {
-    from_port =         "5432"
-    to_port =           "5432"
-    protocol =          "tcp"
-    cidr_blocks =       ["${var.vpc_begin_range}.10.0/24"]
-    ipv6_cidr_blocks =  ["::/0"]
+    from_port = "5432"
```

Comment:
this should be `0` since the application can connect from any port on VM to 5432 of RDS.